docs/documentation_styleguide.rst

```diff
@@ -0,0 +1,234 @@
+========================
+Documentation Styleguide
+========================
+
+.. topic:: Table of contents
+
+    A Table of Contents that should be shown different to admonitation.
```

Comment:
These words are not proper nouns and should not be capitalized in the middle of a sentence.

Also grammar and spelling should be "different from admonition".